### PR TITLE
Removing Incorrect ServiceNow Refrences

### PR DESCRIPTION
**Pull request by tylerezimmerman**
### 🛠 Changes being made

Removed incorrect references to ServiceNow in documentation and scripts. 

### 🏎 Quality check

- [ ] Should this be added to Delinea Marketplace?

- [ ] Has this been tested and verified?

- [ ] Have you included a separate .md documentation for each script submitted?

### CODE REVIEW

File: Scripts/SecretServer/AdobeSign/Discovery/readme.md [hunk 1]
```diff
@@ -245,11 +245,11 @@ See the [Setting the Default PowerShell Credential for a Site](https://docs.deli
 
   
 
-The ServiceNow configuration is now complete. The next step is to run a manual discovery scan.
+The Adobe Sign configuration is now complete. The next step is to run a manual discovery scan.
 
 - Navigate to **Admin | Discovery**
 
-- Click the **Run Discovery Noe** (Dropdon) and select **Run Discovery Now**
+- Click the **Run Discovery Noe** (Dropdown) and select **Run Discovery Now**
```
Comment (jagger) [inline Scripts/SecretServer/AdobeSign/Discovery/readme.md:252]: Now

File: Scripts/SecretServer/EntraId/Discovery/readme.md [hunk 2]
```diff
@@ -238,14 +238,14 @@ See the [Setting the Default PowerShell Credential for a Site](https://docs.deli
 
   
 
-The ServiceNow configuration is now complete. The next step is to run a manual discovery scan.
+The Azure AD / Entra ID configuration is now complete. The next step is to run a manual discovery scan.
 
 - Navigate to **Admin | Discovery**
 
-- Click the **Run Discovery Noe** (Dropdon) and select **Run Discovery Now**
+- Click the **Run Discovery Noe** (Dropdown) and select **Run Discovery Now**
```
Comment (jagger) [inline Scripts/SecretServer/EntraId/Discovery/readme.md:245]: Now

File: Scripts/SecretServer/AdobeSign/Discovery/readme.md (hunk 1 above)
Comment (tylerezimmerman) [reply Scripts/SecretServer/AdobeSign/Discovery/readme.md]: Fixed

File: Scripts/SecretServer/EntraId/Discovery/readme.md (hunk 2 above)
Comment (tylerezimmerman) [reply Scripts/SecretServer/EntraId/Discovery/readme.md]: Fixed

**Review by jagger — APPROVED** [review]

**Pull request merged**